submission/races.go

```diff
@@ -201,6 +203,7 @@ func groupRace(ctx context.Context, chain []ct.ASN1Cert, group *ctpolicy.LogGrou
 			if firstRequested := state.request(logURL, cancel); !firstRequested {
 				return
 			}
+			reqsCounter.Inc(logURL, string(ctfe.AddPreChainName))
```

Comment:
Feels a bit asymmetric having this here -- could it move into `SubmitToLog` ?

Reply:
moved